### PR TITLE
Zookeeper migration changes

### PR DESCRIPTION
**Pull request by dobrerazvan**
### Change log description

_(2-3 concise points about the changes in this PR. When committing this PR, the committer is expected to copy the content of this section to the merge description box)_

### Purpose of the change

_(e.g., Fixes #666, Closes #1234)_

### What the code does

_(Detailed description of the code changes)_

### How to verify it

_(Steps to verify that the changes are effective)_

### CODE REVIEW

File: docker/bin/zookeeperStart.sh [hunk 1]
```diff
@@ -71,23 +81,9 @@ set -e
 
 # Determine if there is an ensemble available to join by checking the service domain
 set +e
-getent hosts $DOMAIN  # This only performs a dns lookup
+nc -v -w2 $CLIENT_HOST $CLIENT_PORT  # This only performs a dns lookup
```
Comment (amuraru) [inline docker/bin/zookeeperStart.sh:101]: why?

Comment (dobrerazvan) [reply docker/bin/zookeeperStart.sh:101]: I belive nc approach is solving the same problem, a bit more elegant, by getting read of the while loop. In the end we want to be able to connect to the SEED node. Am I missing something?

Comment (dobrerazvan) [reply docker/bin/zookeeperStart.sh:101]: This PR is not yet final, was made to get zk expansion work.

**Comment by amuraru** [comment]
@dobrerazvan do we still need this?

**Comment by dobrerazvan** [comment]
@amuraru we have sprint work to make it permanent for the kafka migration track. The PR just shows the differences, we will not gonna merge it this way.

**Pull request closed without merging**